### ISSUE TITLE
TypeScript definitions

### ISSUE THREAD

**Issue by bbqsrc** [catamphetamine/javascript-time-ago#12]
Are there TypeScript definitions defined anywhere for this module? Looking forward to using the Intl polyfill. :)

**Comment by catamphetamine** [comment]
Hi.
I myself don't use TypeScript.
You can submit a pull request with the definitions if you know how to implement them.

**Comment by erfanium** [comment]
Hi I get this error while importing in typescript
```
(node:25584) UnhandledPromiseRejectionWarning: TSError: ⨯ Unable to compile TypeScript:
src/services/telegram/telegram.service.ts(6,21): error TS7016: Could not find a declaration file for module 'javascript-time-ago'. '/home/erfan/Desktop/master/book-keeping/node_modules/javascript-time-ago/index.commonjs.js' implicitly has an 'any' type.
  Try `npm install @types/javascript-time-ago` if it exists or add a new declaration (.d.ts) file containing `declare module 'javascript-time-ago';`

```
I also tried `npm install @types/javascript-time-ago` but it's not work.
what's the solution?

**Comment by catamphetamine** [comment]
This package doesn't include any TypeScript.

**Comment by pbirsinger** [comment]
Using other library because no types

**Comment by catamphetamine** [comment]
@pbirsinger 
> Using other library because no types

As if anyone cares.

**Comment by erikburt** [comment]
I made a PR for types.

**Comment by erikburt** [comment]
Merged ^

**Comment by catamphetamine** [comment]
Update: I've released a new version that has been refactored and many things got renamed.

I've also pushed `index.d.ts.test` in the root directory of the package: I myself am not a TypeScript user so I didn't test if it works. If someone desires, they could briefly check that those typings don't break anything, and if that's the case then we could rename those to `index.d.ts` and ship it in the next version.
